### PR TITLE
This commit renamed all the occurence of MEILISEARCH_HOST to MEILISEA…

### LINKED ISSUE #339: Replace MEILISEARCH_HOST by MEILISEARCH_URL in the docker config
⚠️ This issue is generated, it means the examples and the namings do not necessarily correspond to the language of this repository.
Also, if you are a maintainer, feel free to add any clarification and instruction about this issue.

Sorry if this is already partially/completely implemented, feel free to let me know about the state of this issue in the repo.

Related to https://github.com/meilisearch/integration-guides/issues/214

----

Replace the env var naming from the Dockerfile environment:

- `MEILISEARCH_HOST` to `MEILISEARCH_URL`

Also, look for occurrences inside the docker-compose.yml file.

After, run the tests according to the instructions in the CONTRIBUTING.md.

Also, check for occurrences in the GitHub Actions workflows and inside of the source code.

In the Swift repo:

- [ ] Change the `host` param from `MeiliSearch` to `url`.
- [ ] Replace with `url` all the usages of `MeiliSearch` over the README and code-samples.

### PR DESCRIPTION
**Pull request by zt4ff**
…RCH_URL

# Pull Request

## Related issue
Fixes #339 

## What does this PR do?
- It replace MEILISEARCH_HOST by MEILISEARCH_URL in the docker config

## PR checklist
Please check if your PR fulfils the following requirements:
- [X] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [X] Have you read the contributing guidelines?
- [X] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by brunoocasali** [comment]
Thanks a lot for your contribution @zt4ff :D

File: Tests/MeiliSearchIntegrationTests/Utils.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ private let movies: [Movie] = [
 public let TESTS_TIME_OUT = 10.0
 
 public func currentHost() -> String {
-  ProcessInfo.processInfo.environment["MEILISEARCH_HOST"] ?? "http://localhost:7700"
+  ProcessInfo.processInfo.environment["MEILISEARCH_URL"] ?? "http://localhost:7700"
```
Comment (brunoocasali) [inline Tests/MeiliSearchIntegrationTests/Utils.swift:23]: I can't comment in other places, but can you also change the name of the `host` param? It now should also be `url`.

example:
`client = try MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey")` become `client = try MeiliSearch(url: "http://localhost:7700", apiKey: "masterKey")`

also, in the `code-samples.meilisearch.yml` and over the README :)

**Review by brunoocasali — CHANGES_REQUESTED** [review]

File: Tests/MeiliSearchIntegrationTests/Utils.swift (hunk 1 above)
Comment (brunoocasali) [reply Tests/MeiliSearchIntegrationTests/Utils.swift:23]: Better thinking is not the best way to forward with that. We should deprecate the current param first. Then in another PR and another release version, remove it.

Comment (brunoocasali) [reply Tests/MeiliSearchIntegrationTests/Utils.swift:23]: So, can you "deprecate" the `host` from `MeiliSearch` param to merge this PR?

Comment (brunoocasali) [reply Tests/MeiliSearchIntegrationTests/Utils.swift:23]: Better thinking again, we can do that later. Thanks!

**Review by brunoocasali — APPROVED** [review]
Thanks again for your PR!

bors merge

**Comment by brunoocasali** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Comment by zt4ff** [comment]
Thanks @brunoocasali

**Pull request merged**